api/sql/method_by_id.sql

```diff
@@ -12,6 +12,7 @@ WITH full_thing AS (
     completeness,
     first_author(${articleid}) AS creator,
     last_author(${articleid}) AS last_updated_by,
+    get_authors(${articleid}) as edit_history,
```

Comment:
@paninee it looks like `get_authors` is returning a list of unique authors only, and we need this list to return all edits, not just from unique users.